src/sentry/static/sentry/app/views/settings/components/panelHeader.jsx

```diff
@@ -10,7 +10,9 @@ const getPadding = ({disablePadding, hasButtons}) => css`
 
 const getFlex = ({isFlex}) => (isFlex ? 'display: flex' : '');
 
-const StyledPanelHeader = styled(({disablePadding, ...props}) => <div {...props} />)`
+const StyledPanelHeader = styled(({disablePadding, hasButtons, ...props}) => (
```

Comment:
funny detail about `styled`: if you don't pull a react prop out of the destructuring assignment, it will assign the attribute to the div and react will throw a warning #themoreyouknow